src/berry_mill/mill.py

```diff
@@ -48,7 +47,7 @@ def __init__(self):
                                                             description="berrymill is a root filesystem generator for embedded devices",
                                                             epilog="Have a lot of fun!")
         
-        p.add_argument("-s", "--show-config", action="store_true", help="shows the building configuration")
+        p.add_argument("-s", "--show-config", action="store_true", help="shows the repository configuration")
```

Comment:
Shows the configuration of a repository?

Reply:
of repositories 

Can be multiple right?

Reply:
ah right.